guidelines/enhancement_template.md

```diff
@@ -272,6 +272,40 @@ to implement the design.  For instance,
  > 1. This requires exposing previously private resources which contain sensitive
   information.  Can we do this?
 
+## Security
+
+This section should cover the security impact of the change, if any. For example,
+does it introduce new privileges or capabilities? Does it expose new risks?
+
+### Credentials and RBAC
+
+What credentials will the new component be using if it needs to talk to either the control plane or other components? What RBAC needs to be applied?
+
+Consider the following:
+- Which service accounts will be affected by the change, what permissions will the feature code need?
+- Will end users need to make any security considerations when enabling this feature?
+- What is the scope of the required RBAC?
+  - Are cluster wider permissions required?
+  - Should RBAC be limited to only the namespace(s) where the feature is enabled?
+  - Should RBAC be limited to just the node(s) where the feature is enabled?
```

Comment:
this isn't a capability that exists in kube.

Reply:
Perhaps this one needs rewording, but the sentiment here was similar to the one above. I was thinking about things like the MCD and their new MCN construct. They are having to add new permissions to nodes (not Kubelet) to allow something on each node to update something within the cluster, that could be widely scoped. I want people to think about how to narrow that scope